Scripts/Runtime/Systems/Cache/WorldCache.cs

```diff
@@ -159,6 +160,7 @@ private void ParseSystemGroup(SystemGroupCache parentGroupCache, ComponentSystem
         {
             SystemGroupCache systemGroupCache = (SystemGroupCache)GetOrCreateSystemCache(parentGroupCache, group);
```

Comment:
🐛 
Just realized. You have to make sure you call `group.SortSystems()` before you try to enumerate the update list and infer order. There could be pending sorting that will not get resolved until the next update unless you call this.

Reply:
Fixed.